script/test/cmd/tests.sh

```diff
@@ -177,6 +177,9 @@ convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/examples/docker-com
 # Behavior with -o <dirname>/<filename>
 convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/examples/docker-compose.yml convert -o $TEMP_DIR/output_file -j" "$TEMP_DIR/output_file"
 
+# Test the presence of build args in buildconfig
+convert::expect_success_warning "kompose --provider openshift -f $KOMPOSE_ROOT/script/test/fixtures/buildargs/docker-compose.yml convert --stdout -j" "$KOMPOSE_ROOT/script/test/fixtures/buildargs/openshift-buildargs.json" "INFO Buildconfig using git@github.com:surajnarwade/kompose.git::add-build-args as source."
```

Comment:
also this needs to be investigated how this test is passing even if we don't have function named: `convert::expect_success_warning` but it's named as `convert::expect_success_and_warning`